ci-operator/config/openshift/installer/openshift-installer-master__okd-scos.yaml

```diff
@@ -3,6 +3,10 @@ base_images:
     name: "4.15"
     namespace: origin
     tag: base
+  ocp_4.16_base-rhel9:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
```

Comment:
This looks odd in an SCOS config. Shouldn't it be `origin` and base-... scos9 or something?

Reply:
or maybe this is not required in SCOS at all?

Reply:
Definitely. Nice catch!

Reply:
I see the following images in the `origin` namespace: `4.16:base`, `4.16:base-rhel9`, `4.16:base-7`, `4.16:base-8`. Since the `4.16:base` seems to be centos8/rhel8 based, I'm going to leave the `base-rhel9` in the inputs for now.